src/main/java/org/fcrepo/auth/webac/WebACRolesProvider.java

```diff
@@ -248,8 +243,8 @@ private FedoraResource getBaseVersion(final FedoraResource resource) {
 
         // Transform the effectiveRoles from a Set to a List.
         return effectiveRoles.entrySet().stream()
-            .map(x -> new AbstractMap.SimpleEntry<>(x.getKey(), new ArrayList<>(x.getValue())))
-            .collect(Collectors.toMap(Map.Entry::getKey, Map.Entry::getValue));
+            .map(roleSet -> new AbstractMap.SimpleEntry<>(roleSet.getKey(), new ArrayList<>(roleSet.getValue())))
```

Comment:
If I didn't think that method signature was going to change, yes, I would prefer to use guava here. But because in the next week or two I plan to change that method signature to accept a `Collection`, I'd probably just keep this as it is.